### PR TITLE
Update Package.swift for CI.

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

File: .github/workflows/functions.yml [hunk 1]
```diff
@@ -45,6 +45,8 @@ jobs:
     # Don't run on private repo unless it is a PR.
     if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
     runs-on: macos-11
+    env:
+      SWIFTCI_USE_LOCAL_DEPS: 1
```
Comment (paulb777) [inline .github/workflows/functions.yml:49]: Let's namespace this with perhaps `FIREBASECI_` and its more use latest than local_deps, so how about `FIREBASECI_USE_LATEST_GOOGLEAPPMEASUREMENT`

Comment (granluo) [reply .github/workflows/functions.yml]: The [sed](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/setup_spm_tests.sh#L24) could be removed after this is submitted. We can do it in another PR.

**Review by paulb777 — COMMENTED** [review]
I suspect that spm.yml may also need the same `env` change

**Review by paulb777 — APPROVED** [review]
OK to merge this and follow up with the sed script and spm.yml separately

**Comment by granluo** [comment]
> OK to merge this and follow up with the sed script and spm.yml separately

The change will not fail the current workflows with `sed`, so we can have separate PRs for that.

**Pull request merged**